pages/common/virsh-pool-define-as.md

```diff
@@ -0,0 +1,9 @@
+# virsh-pool-define-as
+
+> Create a configuration file in `/etc/libvirt/storage` for a persistent virtual machine storage pool from the provided arguments.
+> See also: `virsh`.
+> More information: <https://manned.org/virsh>.
+
+- Create the configuration file for a storage pool called pool_name using `/var/vms` as the underlying storage system (use `virsh pool-build` to create the underlying storage system if it doesn't exist and use `virsh pool-start` to start the inactive pool):
```

Comment:
This is a little too long and can be shortened very easily

Reply:
Done. It didn't get caught by the version of tldrl I'm using.